exotations/task_maps/task_map/src/task_map_py.cpp

```diff
@@ -38,6 +39,9 @@ PYBIND11_MODULE(task_map_py, module)
     effAxisAlignment.def("getDirection", &EffAxisAlignment::getDirection);
     effAxisAlignment.def("setDirection", &EffAxisAlignment::setDirection);
 
+    py::class_<Point2Line, std::shared_ptr<Point2Line>, TaskMap> point2Line(module, "Point2Line");
+    point2Line.def("setEndPoint", &Point2Line::setEndPoint);
```

Comment:
Change this to a property with getter and setter, i.e., use ``def_property`` (we have been inconsistent in the past but now starting to consolidate).

Reply:
Ok, so as I understand from [here](https://pybind11.readthedocs.io/en/stable/classes.html#instance-and-static-fields) L43 (and L39/40 for that matter) should be:
```
effAxisAlignment.def_property("direction", &EffAxisAlignment::getDirection, &EffAxisAlignment::setDirection);

point2Line.def_property("end_point", &Point2Line::getEndPoint, &Point2Line::setEndPoint);
```

Reply:
Specify bot setter and getter

Reply:
Yes, your understanding here is correct